invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -86,22 +88,28 @@ if (sidebarCommunitiesManageDiv) {
   const pendingCommunitiesSearchConfig =
     sidebarCommunitiesManageDiv.dataset.pendingCommunitiesSearchConfig;
   ReactDOM.render(
-    <CommunitiesManagement
-      userCommunitiesMemberships={JSON.parse(
-        sidebarCommunitiesManageDiv.dataset.userCommunitiesMemberships
-      )}
-      recordCommunityEndpoint={
-        sidebarCommunitiesManageDiv.dataset.recordCommunityEndpoint
-      }
-      recordUserCommunitySearchConfig={JSON.parse(
-        sidebarCommunitiesManageDiv.dataset.recordUserCommunitySearchConfig
-      )}
-      canManageRecord={JSON.parse(sidebarCommunitiesManageDiv.dataset.canManageRecord)}
-      recordCommunitySearchConfig={recordCommunitySearchConfig}
-      permissions={JSON.parse(sidebarCommunitiesManageDiv.dataset.permissions)}
-      searchConfig={JSON.parse(pendingCommunitiesSearchConfig)}
-      record={JSON.parse(recordCitationAppDiv.dataset.record)}
-    />,
+    <OverridableContext.Provider value={overriddenComponents}>
```

Comment:
sorry I am a bit rusty on the implementation, could you remind me why the context provider is needed for adding the overridable component?

Reply:
The context provider creates a context that gives its children access to the overridestore. Without the OverridableContext, the Overriadable does not know by which component it should be overridden.